include/ignition/rendering/base/BaseArrowVisual.hh

```diff
@@ -35,9 +35,9 @@ namespace ignition
 
       public: virtual ~BaseArrowVisual();
 
-      public: virtual VisualPtr Head() const;
+      public: virtual VisualPtr Head() const override;
 
-      public: virtual VisualPtr Shaft() const;
+      public: virtual VisualPtr Shaft() const override;
 
       protected: virtual void Init();
```

Comment:
since we added `override` to other functions in this class, can you add the `override` keyword to this function to fix the homebrew warning:
https://build.osrfoundation.org/job/ignition_rendering-ci-pr_any-homebrew-amd64/675/warnings10Result/new/